src/components/__snapshots__/Calculator.test.js.snap

```diff
@@ -0,0 +1,142 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
```

Comment:
- [OPTIONAL] According to the project requirements, There should be the usage of Jest snapshots for testing all the components. shower you only have Calculator, Home, and Quote snapshots testing. Please you can consider also adding the Jest snapshots for testing the `Nav` component.